README.md

```diff
@@ -36,6 +36,18 @@
 ## Node Version
 Node Versions below [14](https://nodejs.org) are not supported due to package incompatibility and degraded performance.
 
+## Running a game server 
+
+- Import you game.json file into /taro/src/ directory
+
+To run the game server, execute the following command:
+```
+npm run server
+```
+
```

Comment:
Why are we adding this when we still have the ability to fetch game.json from api?